src/templates/amfcfg.conf.j2

```diff
@@ -0,0 +1,164 @@
+configuration:
+  amfDBName: {{ amf_database_name }}
+  amfName: AMF
+  debugProfilePort: 5001
+  enableDBStore: false
+  enableSctpLb: false
+  kafkaInfo:
+    brokerPort: 9092
+    brokerUri: sd-core-kafka-headless
+    topicName: sdcore-data-source-amf
+  mongodb:
+    name: {{ default_database_name }}
+    url: {{ database_url }}
+  networkFeatureSupport5GS:
+    emc: 0
+    emcN3: 0
+    emf: 0
+    enable: true
+    imsVoPS: 0
+    iwkN26: 0
+    mcsi: 0
+    mpsi: 0
+  networkName:
+    full: Aether5G
+    short: Aether
+  nfKafka:
+    enable: false
+    topic: sdcore-nf-data-source
+    urls:
+    - sd-core-kafka-headless:9092
+  ngapIpList:
+  - 0.0.0.0
+  ngappPort: 38412
+  non3gppDeregistrationTimerValue: 3240
+  nrfUri: {{ nrf_url }}
+  plmnSupportList:
```

Comment:
Same here, ideally we would remove this since it's not going to be used

Reply:
Everything that is specified in [simapp/webui](https://github.com/gruyaume/omec-simapp-operator/blob/main/src/files/default_config.yaml) (like mcc, mnc) should not be here as it makes it confusing for us. Reading this makes it seems that we are hardcoding the mcc to 208 but the reality is that we're not, we're reading the config from the webui address by using the `MANAGED_BY_CONFIG_POD` environment variable.

Reply:
I kept the parts that are not passed by the[ default config of simapp](https://github.com/gruyaume/omec-simapp-operator/blob/main/src/files/default_config.yaml). Please take a look and let me know if that's what you meant :)
I'd also be interested if you see any parts that should be passed as parameters.